### PR TITLE
Update document to new template

### PR DESCRIPTION
**Pull request by gcomoretto**
The DMTR-102 has been updated using the latest template.

Please review it and approve before merging it to master.

### CODE REVIEW

File: DMTR-102.tex [hunk 1]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
 \subsection{Objectives}
 \label{sect:objectives}
 
-Testing for the raw image from the DAQ to being written to the archive service at the LDF
-This test activity demonstrates the successful execution of the major components of the storing of raw image files from the DAQ to the DBB and then visualized through the LSST science platform (LSP).   This is at a very small scale and it to test the components of a proof of concept that they all work together.  
+Testing for the raw image from the DAQ to being written to the archive
```
Comment (leannep) [inline DMTR-102.tex]: Test that raw images from the DAQ can be written to the archive service at the LDF

File: DMTR-102.tex [hunk 2]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
+service at the LDF\\
+This test activity demonstrates the successful execution of the major
```
Comment (leannep) [inline DMTR-102.tex]: This test activity will demonstrate the successful execution of the major components required to transfer raw image files from the DAQ to the DBB and subsequently visualize them via the LSST Science Platform(LSP).  This is a small scale proof-of-concept test designed to demonstrate that the key components can interoperate successfully.

File: DMTR-102.tex [hunk 3]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
+components of the storing of raw image files from the DAQ to the DBB and
+then visualized through the LSST science platform (LSP). ~ This is at a
+very small scale and it to test the components of a proof of concept
+that they all work together. ~\\[2\baselineskip]It will demonstrate
```
Comment (leannep) [inline DMTR-102.tex]: Specifically, it will demonstrate that:

File: DMTR-102.tex [hunk 4]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
 \begin{itemize}
-\item Data can be transferred into the lsst data facility (LDF).   
-\item The raw data images can be ingested by the DBB along with some metadata about the image and providence.  
-\item The data files  can then be retrieved by the LSP
+\tightlist
+\item
+  Data can be transferred into the lsst data facility (LDF). ~~
+\item
+  The raw data images can be ingested by the DBB along with some
+  metadata about the image and providence. ~
```
Comment (leannep) [inline DMTR-102.tex]: ...along with a subset of image and providence metadata

File: DMTR-102.tex [hunk 5]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
+\item
+  The data files ~can then be retrieved by the LSP
```
Comment (leannep) [inline DMTR-102.tex]: The transferred data files can then be retrieved via the LSP.

File: DMTR-102.tex [hunk 6]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
+
 
-The overall test plan for the LSST Data Management system is described in LDM-503.
-This document specifically refers to the milestone LDM-503-8b, which tests the raw image archiving for a CCOB device.  
 
 \subsection{System Overview}
 \label{sect:systemoverview}
 
-The raw image archiving system that is part of the LSST Data Management system is responsible for the data being stored into the archive environment at the LDF from various sources.  It will depend on what DAQ system (spectragraph, comcam, lsstcam, CCOB... others) as to where the data is located originally, and then how it will be sent to NCSA (LDF) .    The tests begin with data somewhere that needs to be stored at the archive enviornment at LDF and ingested into the DBB.   The LDM-503-5 milestone is limited to the raw image archive from a CCOB device, not all types of devices and is limited in scope.  
+The raw image archiving system that is part of the LSST Data Management
```
Comment (leannep) [inline DMTR-102.tex:73]: The raw image archiving system,  which part of the LSST Data Management, system is responsible for the data being stored in the archive environment at the LDF from various sources.

File: DMTR-102.tex [hunk 7]
```diff
@@ -1,40 +1,36 @@
...
 
+\providecommand{\tightlist}{
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+
 \begin{document}
 
-\def\milestoneName{Raw Image Archiving}
-\def\milestoneId{LDM-503-8b}
-\def\product{LSST Raw Image Archiving System}
+\def\milestoneName{Raw image archiving for data from ccob device}
```
Comment (jdswinbank) [inline DMTR-102.tex]: CCOB, not ccob.

According to LDM-503, this is  “Small Scale CCOB Data Access”. Why is the milestone name changing here? If it needs to change, we should update P6 and the baselined documentation, too.

File: DMTR-102.tex [hunk 8]
```diff
@@ -1,40 +1,36 @@
...
+\def\milestoneId{503-8b}
```
Comment (jdswinbank) [inline DMTR-102.tex]: `503-08b` according to P6.

File: DMTR-102.tex [hunk 9]
```diff
@@ -1,40 +1,36 @@
...
 
-\setDocCurator{Michelle Butler}
-\setDocUpstreamLocation{\url{https://github.com/lsst-dm/\lsstDocType-\lsstDocNum}}
-\setDocUpstreamVersion{\vcsrevision}
+\input history_and_info.tex
 
 
 \setDocAbstract{
-This is the test plan and report for \milestoneId{} (\milestoneName{}), an LSST DM level 1 milestone pertaining to the \product{}.
+This is the test plan and report for \milestoneId{} (\milestoneName{}), an LSST level 2 milestone pertaining to the \product{} Subsystem.
```
Comment (jdswinbank) [inline DMTR-102.tex]: Archiving isn't a subsystem. See comments on DMTR-91 (https://github.com/lsst-dm/DMTR-91/pull/1) — many of them apply here too.

File: DMTR-102.tex [hunk 10]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
 \subsection{Objectives}
 \label{sect:objectives}
 
-Testing for the raw image from the DAQ to being written to the archive service at the LDF
-This test activity demonstrates the successful execution of the major components of the storing of raw image files from the DAQ to the DBB and then visualized through the LSST science platform (LSP).   This is at a very small scale and it to test the components of a proof of concept that they all work together.  
+Testing for the raw image from the DAQ to being written to the archive
+service at the LDF\\
+This test activity demonstrates the successful execution of the major
+components of the storing of raw image files from the DAQ to the DBB and
+then visualized through the LSST science platform (LSP). ~ This is at a
```
Comment (jdswinbank) [inline DMTR-102.tex]: Why `~`?

File: DMTR-102.tex [hunk 11]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
+very small scale and it to test the components of a proof of concept
```
Comment (jdswinbank) [inline DMTR-102.tex]: “it to test the components of a proof of concept that they all work together” isn't really English!

File: DMTR-102.tex [hunk 12]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
+that they all work together. ~\\[2\baselineskip]It will demonstrate
 
-It will demonstrate
 \begin{itemize}
-\item Data can be transferred into the lsst data facility (LDF).   
-\item The raw data images can be ingested by the DBB along with some metadata about the image and providence.  
-\item The data files  can then be retrieved by the LSP
+\tightlist
+\item
+  Data can be transferred into the lsst data facility (LDF). ~~
```
Comment (jdswinbank) [inline DMTR-102.tex]: So many spurious `~`s! I guess they are harmless, though.

File: DMTR-102.tex (hunk 4 above)
Comment (jdswinbank) [inline DMTR-102.tex]: Provenance? 😆

File: DMTR-102.tex [hunk 13]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
 \label{sect:systemoverview}
 
-The raw image archiving system that is part of the LSST Data Management system is responsible for the data being stored into the archive environment at the LDF from various sources.  It will depend on what DAQ system (spectragraph, comcam, lsstcam, CCOB... others) as to where the data is located originally, and then how it will be sent to NCSA (LDF) .    The tests begin with data somewhere that needs to be stored at the archive enviornment at LDF and ingested into the DBB.   The LDM-503-5 milestone is limited to the raw image archive from a CCOB device, not all types of devices and is limited in scope.  
+The raw image archiving system that is part of the LSST Data Management
+system is responsible for the data being stored into the archive
+environment at the LDF from various sources. ~It will depend on what DAQ
+system (spectragraph, comcam, lsstcam, CCOB\ldots{} others) as to where
+the data is located originally, and then how it will be sent to NCSA
+(LDF) . ~ ~The tests begin with data somewhere that needs to be stored
+at the archive enviornment at LDF and ingested into the DBB. ~ The
```
Comment (jdswinbank) [inline DMTR-102.tex]: Typo “enviornment”.

File: DMTR-102.tex [hunk 14]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
+LDM-503-5 milestone is limited to the raw image archive from a CCOB
+device, not all types of devices and is limited in scope.
```
Comment (jdswinbank) [inline DMTR-102.tex]: LDM-503-05 is “alert distribution validation”; I don't think it's correct here.

File: DMTR-102.tex [hunk 15]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
-\label{sect:appdocs}
-\addtocounter{table}{-1}
+\subsection{Applicable Documents}\label{applicable-documents}
 
-\begin{tabular}[htb]{l l}
-\citeds{LDM-294} & LSST DM Project Management Plan\\
-\citeds{LDM-503} & DM Test Plan\\
-\citeds{LDM-538} & DM Raw Image Archiving Service Test Specification \\
-\end{tabular}
+\citeds{LDM-294} LSST DM Project Management Plan\\
```
Comment (jdswinbank) [inline DMTR-102.tex]: “Data Management Organization and Management”

File: DMTR-102.tex [hunk 16]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
 \begin{itemize}
-\item Test Plan (LVV-P10) Owner: Michelle Butler
-\item Test Cycle (LVV-C8) Owner: Unassigned
-\item LVV-T284 Tester: Michelle Butler
-\item Additional Test Personel involved: None 
+\item Test Plan (LVV-P10) owner: Michelle Butler
+\item Test Cycles:
+\begin{itemize}
+  \item LVV-C8 owner: 
+    Undefined
```
Comment (jdswinbank) [inline DMTR-102.tex:146]: Should have an owner.

File: DMTR-102.tex [hunk 17]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
-\label{sect:overallassessment}
+    This test will check:
 
+\begin{itemize}
+\tightlist
+\item
+  The successful integration of the Pathfinder components with the CCOB;
+\item
+  That the file can then be ingested into the DBB and be retrieved for
+  further analysis;
```
Comment (jdswinbank) [inline DMTR-102.tex]: Should not end in a `;`.

File: DMTR-102.tex [hunk 18]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
+
+    Detailed step results:
+
+    \begin{longtable}{p{1cm}p{2cm}p{13cm}}
+    \hline
+    {Step} & \multicolumn{2}{c}{Description, Results and Status}\\ \hline
+      1 & Description &
+
+      \begin{minipage}[t]{13cm}{\footnotesize
+      CCOB device directs a human to where a raw file is wanted to be stored
```
Comment (jdswinbank) [inline DMTR-102.tex:279]: How does the test start? Will the CCOB spontaneously light up and direct the human to a file? Or does somebody need to take data on the CCOB? Or press a button? Or...?

File: DMTR-102.tex [hunk 19]
```diff
@@ -44,183 +40,377 @@ \section{Introduction}
...
+      & \begin{minipage}[t]{2cm}{Actual\\ Result}\end{minipage}   & 
+      \begin{minipage}[t]{13cm}{\footnotesize
+      
+      \vspace{\dp0}
+      } \end{minipage} \\
+      \\ \cdashline{2-3}
+
+      & Status          & Not Executed \\ \hline
+
+      5 & Description &
```
Comment (jdswinbank) [inline DMTR-102.tex:399]: This step is empty.

File: history_and_info.tex [hunk 20]
```diff
@@ -0,0 +1,9 @@
+% Most recent last
+\setDocChangeRecord{
+\addtohist{}{2018-10-31}{Test plan Update}{Butler, Comoretto}
```
Comment (jdswinbank) [inline history_and_info.tex]: Doesn't want a capital “U”.

File: meta.tex [hunk 21]
```diff
@@ -1,5 +1,5 @@
 % GENERATED FILE -- edit this in the Makefile
```
Comment (jdswinbank) [inline meta.tex]: `meta.tex` should not be committed to the repository.

File: DMTR-102.tex (hunk 7 above)
Comment (gcomoretto) [reply DMTR-102.tex]: Considering that the test plan name corresponds to the milestones, in order to get the document correct we need:
- update LVV-P10 with the proper milestone name (@mbutlerncsa )
- update docsteady, in this moment is capitalizing the string, and this will made low case the rest of the string (update on issue (DM-16678)
- update the procedure: in  case the Test plan correspond to a milestone, use the milestone as name of the test plan (new issue on me)

File: DMTR-102.tex (hunk 8 above)
Comment (gcomoretto) [reply DMTR-102.tex]: To be updated in LVV-P10 name (@mbutlerncsa )

File: DMTR-102.tex (hunk 9 above)
Comment (gcomoretto) [reply DMTR-102.tex]: The document folder structure need to be adherent to the document tree.

File: DMTR-102.tex (hunk 10 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This is an extra space in the Jira LVV-P10 Objective field. That pandoc render as a ~. If this extra space is not desided, has to be removed from LVV-P10.

File: DMTR-102.tex (hunk 11 above)
Comment (gcomoretto) [reply DMTR-102.tex]: Please update LVV-P10 (@mbutlerncsa )

File: DMTR-102.tex (hunk 17 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This need to be changed in the test case LVV-T284.

File: DMTR-102.tex (hunk 18 above)
Comment (gcomoretto) [reply DMTR-102.tex:279]: This need to be changed in the test case LVV-T284.

File: DMTR-102.tex (hunk 19 above)
Comment (gcomoretto) [reply DMTR-102.tex:399]: This has already been corrected in the test case.
The test player need to be updated to the latest revision of the test case.

File: history_and_info.tex (hunk 20 above)
Comment (gcomoretto) [reply history_and_info.tex]: My fault, I correct the history file.

File: meta.tex (hunk 21 above)
Comment (gcomoretto) [reply meta.tex]: I remove it and I add to the .gitignore

File: DMTR-102.tex (hunk 11 above)
Comment (leannep) [reply DMTR-102.tex]: I already provided a suggested rewording above for the English

File: DMTR-102.tex (hunk 12 above)
Comment (leannep) [reply DMTR-102.tex]: I think this is in the autogeneration script - there should be one parse to fix them all at the source

File: DMTR-102.tex [hunk 22]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
-\label{sect:references}
+This document was generated from Jira, obtaining the relevant information from the 
+\href{https://jira.lsstcorp.org/secure/Tests.jspa#/testPlan/LVV-P10}{LVV-P10}
+~Jira Test Plan and related Test Cycles (
+  \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCycle/LVV-C8}{LVV-C8}
+).
 
-\renewcommand{\refname}{}
-\bibliography{lsst,refs,books,refs_ads}
+%The following general sections are completed before the start of the test activity.
```
Comment (leannep) [inline DMTR-102.tex]: I know it is commented but if it gets uncommented change to : The following general sections shall be completed before the start of the test activity.

File: DMTR-102.tex [hunk 23]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
 
-\subsection{Document Overview}
-\label{sect:docoverview}
+Section \ref{sect:intro} provides an overview of the test campaign, the system under test (\product{}), the applicable documentation, and explains how this document is organized.
+Section \ref{sect:configuration}  describes the configuration used for this test.
+Section \ref{sect:personnel} lists all the individuals involved and describes their roles.
```
Comment (leannep) [inline DMTR-102.tex]: I would change the order of roles and individuals in the personnel section description. For any given test campaign, we should first define the necessary roles and then assign individuals. Suggest: 'describes the necessary roles and lists the individuals assigned to them'

File: DMTR-102.tex [hunk 24]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
-\label{sect:swconf}
+\subsection{Verification Environment}
+\label{sect:hwconf}
+  LDF center at NCSA. ~ The archive service or data back bone.~~
 
-The DBB version is 1.0, and can be found in the github repository when it has been fully developed.   The current DBB is a script for scraping data from the header file and getting the file name and the metadata required along with providence.   At the time of this writing, the DBB use cases and requirements are just being written.    The database component is Oracle and the ingest file systems and output file systems for DBB are in GPFS at the LDF.  
 
-\subsection{Entry Criteria}
+  \subsection{Entry Criteria}
+  Data that needs to be stored from a CCOB or DAQ
```
Comment (leannep) [inline DMTR-102.tex:127]: Raw image data from a CCOB or DAQ available for archiving.

File: DMTR-102.tex [hunk 25]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
 
-Data that needs to be stored from a CCOB or DAQ.
 
-\subsection{Exit Criteria}
+  \subsection{Exit Criteria}
+  Data is transferred in proper form and can be viewed on LSP after being
+transferred to LDF and ingested into DBB.~~
```
Comment (leannep) [inline DMTR-102.tex]: Data are correctly transferred to the LDF and ingested into the DBB,  and can be subsequently viewed on the LSP.

File: DMTR-102.tex [hunk 26]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
+  503-8b raw image archiving for CCOB cycle\\
+  Status: Not Executed
 
-            \vspace{\dp0}
-            } \end{minipage} \\ \cline{2-3}
+  Test cycle for the raw image archiving of data from a CCOB device. ~This
+data needs to be stored at the LDF and ingested so that it can be looked
+at by scientists. ~ The data needs to be transferred to the LDF and then
+ingested into the archive service so that it can be retrieved through a
+butler to be displayed for further use. ~ This should be tested with a
```
Comment (leannep) [inline DMTR-102.tex:204]: ... so that it can be inspected by scientists.

This should be first tested manually by having a person move the data and checking that all the components of the system work.

File: DMTR-102.tex [hunk 27]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
+human involved at first for moving the data and making sure all the
+pieces work. ~ It should also be tested for when things are working more
+as a service, and without a human involved for an automatic data archive
+process. ~
```
Comment (leannep) [inline DMTR-102.tex]: Following manual testing, the integrated system should also be tested as an automated data archive service, without the need for manual intervention.

File: DMTR-102.tex [hunk 28]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
 
-        \\ \midrule
-    \end{longtable}
 
+  \subsubsection{Software Version/Baseline}
+    The DBB version is 1.0, and can be found in the github repository when
```
Comment (leannep) [inline DMTR-102.tex]: The DBB version used in this test campaign is 1.0, ....

File: DMTR-102.tex [hunk 29]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
+it has been fully developed. ~ The current DBB is a script for scraping
+data from the header file and getting the file name and the metadata
+required along with providence. ~ At the time of this writing, the DBB
```
Comment (leannep) [inline DMTR-102.tex]: providence -> provenance

File: DMTR-102.tex (hunk 9 above)
Comment (jdswinbank) [reply DMTR-102.tex]: I think that's fine in the long term, but:

- In the short term, the document tree isn't stable.
- They still aren't subsystems! They are products (or services, or deliverables, or whatever) of the Data Management Subsystem.

Comment (gcomoretto) [reply DMTR-102.tex]: I get the point now. I used the same working as in the manually written DMTRs. I will use the word "deliverable" instead of subsystem.

Comment (gcomoretto) [reply DMTR-102.tex]: OK, I rephrase it in the template.

File: DMTR-102.tex [hunk 30]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
+use cases and requirements are just being written. ~ ~The database
+component is Oracle and the ingest file systems and output file systems
+for DBB are in GPFS at the LDF.~~
 
 
-\newpage
+  \subsubsection{Configuration}
+    The configuration for this test is the data is located in a file system
+somewhere. ~It is transferred to NCSA into a the LSST GPFS file system.
```
Comment (leannep) [inline DMTR-102.tex]: It is transferred to the LSST GPFS file system at NCSA.

File: DMTR-102.tex [hunk 31]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
+~ Files that are needed in the DBB are moved to the DBB ingest file
+system. ~ the data backbone gathers the info from the headers and file
```
Comment (leannep) [inline DMTR-102.tex]: **The** data backbone gathers the necessary information from the headers and file name. **It**

File: DMTR-102.tex [hunk 32]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
+name. ~it inserts rows into the DBB consolidated DB environment and
+moves the file to the DBB file systems that are used by the LSP to once
+again find the raw image file to be viewed/displayed for further
+analysis. ~
```
Comment (leannep) [inline DMTR-102.tex]: raw image file to be inspected for further analysis. ~

File: DMTR-102.tex (hunk 9 above)
Comment (gcomoretto) [reply DMTR-102.tex]: Done

File: DMTR-102.tex [hunk 33]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
+test case in Jira.
 
-\subsubsection{Overall Assessment}
-\label{sect:overallassessment}
+    This test will check:
 
+\begin{itemize}
+\tightlist
+\item
+  The successful integration of the Pathfinder components with the CCOB;
```
Comment (leannep) [inline DMTR-102.tex]: 'Pathfinder components' has not been previously defined.

File: DMTR-102.tex [hunk 34]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
+      } \end{minipage} \\
+      \\ \cdashline{2-3}
+
+      & Status          & Not Executed \\ \hline
+
+      4 & Description &
+
+      \begin{minipage}[t]{13cm}{\footnotesize
+      The LSP can review and use the CCOB raw data file that was stored
+originally somewhere else such as slac
```
Comment (leannep) [inline DMTR-102.tex:373]: The 'LSP' does not actively do anything, a human user does things via the LSP. I would rephrase as: 'A user can retrieve and inspect the transferred  CCOB raw data file.'

File: DMTR-102.tex (hunk 16 above)
Comment (leannep) [reply DMTR-102.tex:146]: Should we not also have a 'Verifier/Tester' role. Mention is made several times  in this test plan about the need to have a human retrieve and inspect the raw images transferred from the CCOB to NCSA as the final acceptance criteria.

File: DMTR-102.tex (hunk 18 above)
Comment (leannep) [reply DMTR-102.tex:279]: I don't understand 'a file is wanted to be stored in the DBB' - I smell a cut-and-paste error

File: DMTR-102.tex [hunk 35]
```diff
@@ -44,183 +40,378 @@ \section{Introduction}
...
+      } \end{minipage} \\
+      \\ \cdashline{2-3}
+
+      & Status          & Not Executed \\ \hline
+
+      2 & Description &
+
+      \begin{minipage}[t]{13cm}{\footnotesize
+      Move the data from the transferred directory into the DBB foreign file
+ingest file system. ~\\[2\baselineskip]
```
Comment (leannep) [inline DMTR-102.tex]: What is the 'transferred directory' - do you mean the directory on the filesystem at NCSA into which the image files were transferred?

**Review by leannep — CHANGES_REQUESTED** [review]

File: DMTR-102.tex (hunk 12 above)
Comment (gcomoretto) [reply DMTR-102.tex]: The autogeneration script is preserving extra spaces when provided. They are harmless. You can edit the text in the test cases/test plan/test cycles/test results in order to get rid of them.

File: DMTR-102.tex (hunk 4 above)
Comment (gcomoretto) [reply DMTR-102.tex]: To correct in LVV-P10, Objective field.

File: DMTR-102.tex (hunk 13 above)
Comment (gcomoretto) [reply DMTR-102.tex]: To correct in LVV-P10 System Overview field.

File: DMTR-102.tex (hunk 14 above)
Comment (gcomoretto) [reply DMTR-102.tex]: same as above.

File: DMTR-102.tex (hunk 15 above)
Comment (gcomoretto) [reply DMTR-102.tex]: same as above. 
LDM-538 shall be kept as applicable document, and test cases shall be review there, before including them in a test plan like this.

File: DMTR-102.tex (hunk 16 above)
Comment (gcomoretto) [reply DMTR-102.tex:146]: The tester is specified for each test case.

File: DMTR-102.tex (hunk 7 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: done.  Updated LVV-P10

File: DMTR-102.tex (hunk 8 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: Done

File: DMTR-102.tex (hunk 7 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: fixed

File: DMTR-102.tex (hunk 8 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: fixed

File: DMTR-102.tex (hunk 9 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: thanks.  I read through DMTR-91.   :-)   I changed wording to reflect that advice.

File: DMTR-102.tex (hunk 1 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This need to be changed in the test plan LVV-P10.

File: DMTR-102.tex (hunk 2 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This need to be updated in the test plan LVV-P10

File: DMTR-102.tex (hunk 3 above)
Comment (gcomoretto) [reply DMTR-102.tex]: To update in the test plan LVV-P10

File: DMTR-102.tex (hunk 4 above)
Comment (gcomoretto) [reply DMTR-102.tex]: To update in the test plan LVV-P10

File: DMTR-102.tex (hunk 5 above)
Comment (gcomoretto) [reply DMTR-102.tex]: To be updated in the LVV-P10 test plan.

File: DMTR-102.tex (hunk 6 above)
Comment (gcomoretto) [reply DMTR-102.tex:73]: To be updated in the LVV-P10

File: DMTR-102.tex (hunk 22 above)
Comment (gcomoretto) [reply DMTR-102.tex]: Commented line removed from the template.

File: DMTR-102.tex (hunk 23 above)
Comment (gcomoretto) [reply DMTR-102.tex]: Template updated.

File: DMTR-102.tex (hunk 24 above)
Comment (gcomoretto) [reply DMTR-102.tex:127]: This text need to be updated in the test plan LVV-P10.

File: DMTR-102.tex (hunk 25 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This text need to be updated in the test plan LVV-P10.

File: DMTR-102.tex (hunk 26 above)
Comment (gcomoretto) [reply DMTR-102.tex:204]: This text need to be updated in the test cycle LVV-C8.

File: DMTR-102.tex (hunk 27 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This text need to be updated in the test cycle LVV-C8

File: DMTR-102.tex (hunk 28 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This text need to be updated in the test cycle LVV-C8

File: DMTR-102.tex (hunk 29 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This text need to be updated in the test cycle LVV-C8

File: DMTR-102.tex (hunk 31 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This text need to be updated in the test cycle LVV-C8.

File: DMTR-102.tex (hunk 32 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This text need to be updated in the test cycle LVV-C8.

File: DMTR-102.tex (hunk 30 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This text need to be updated in the test cycle LVV-C8

File: DMTR-102.tex (hunk 33 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This text need to be updated in the test case LVV-T284.

File: DMTR-102.tex (hunk 35 above)
Comment (gcomoretto) [reply DMTR-102.tex]: This text need to be updated in the test case LVV-T284.

File: DMTR-102.tex (hunk 34 above)
Comment (gcomoretto) [reply DMTR-102.tex:373]: This text need to be updated in the test case LVV-T284.

File: DMTR-102.tex (hunk 11 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: took Leanne wording.   
done.

File: DMTR-102.tex (hunk 12 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: I looked for as many as I could find.  done.

File: DMTR-102.tex (hunk 4 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: corrected spelling

File: DMTR-102.tex (hunk 13 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: fixed.

File: DMTR-102.tex (hunk 14 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: swinbank: you are correct. It was 503-4-- sorry

File: DMTR-102.tex (hunk 15 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: fixed.

File: DMTR-102.tex (hunk 16 above)
Comment (mbutlerncsa) [reply DMTR-102.tex:146]: I put a tester on each test that was human done.

File: DMTR-102.tex (hunk 17 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: done

File: DMTR-102.tex (hunk 10 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: I tried to find as many extra spaces that I could find.

File: DMTR-102.tex (hunk 18 above)
Comment (mbutlerncsa) [reply DMTR-102.tex:279]: fixed wording

File: DMTR-102.tex (hunk 1 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: fixed

File: DMTR-102.tex (hunk 2 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: done.

File: DMTR-102.tex (hunk 3 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: done.

File: DMTR-102.tex (hunk 4 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: done.

File: DMTR-102.tex (hunk 5 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: done.

File: DMTR-102.tex (hunk 33 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: changed txt

File: DMTR-102.tex (hunk 34 above)
Comment (mbutlerncsa) [reply DMTR-102.tex:373]: fixed

File: DMTR-102.tex (hunk 35 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: done.

File: DMTR-102.tex (hunk 6 above)
Comment (mbutlerncsa) [reply DMTR-102.tex:73]: done.

File: DMTR-102.tex (hunk 24 above)
Comment (mbutlerncsa) [reply DMTR-102.tex:127]: done

File: DMTR-102.tex (hunk 25 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: fixed

File: DMTR-102.tex (hunk 26 above)
Comment (mbutlerncsa) [reply DMTR-102.tex:204]: fixed

File: DMTR-102.tex (hunk 27 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: done

File: DMTR-102.tex (hunk 28 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: done

File: DMTR-102.tex (hunk 29 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: fixed

File: DMTR-102.tex (hunk 30 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: fixed

File: DMTR-102.tex (hunk 31 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: fixed

File: DMTR-102.tex (hunk 32 above)
Comment (mbutlerncsa) [reply DMTR-102.tex]: fixed

**Review by leannep — APPROVED** [review]
